smithy-aws-cloudformation/src/main/java/software/amazon/smithy/aws/cloudformation/schema/model/Property.java

```diff
@@ -87,7 +106,7 @@ public Schema getSchema() {
 
     public static final class Builder implements SmithyBuilder<Property> {
         private boolean insertionOrder = false;
-        private final List<String> dependencies = new ArrayList<>();
+        private final List<String> dependencies = new LinkedList<>();
```

Comment:
An `ArrayList` is insertion ordered, why a `LinkedList` here?

Reply:
Artifact from a previous change - reverted.